JustSaying.AwsTools.IntegrationTests/WhenSettingUpMultipleHandlers.cs

```diff
@@ -97,5 +99,18 @@ public void CreateQueueCalledOnce()
         {
             Assert.That(proxyAwsClientFactory.Counters["CreateQueue"][queueName].Count, Is.EqualTo(1));
         }
+
+        [Test]
+        public void SetQueueAttributesCalledOnceWithBasicAttributes()
+        {
+            var allSetQueueAttributesCalls = proxyAwsClientFactory.Counters["SetQueueAttributes"]
```

Comment:
This is ugly but not sure how to do it better. There are many calls to `SetQueueAttributes()` and I'm only interested in the one that happens immediately after the queue is created